myProject/my_app/analysis_options.yaml

```
@@ -0,0 +1,5 @@
+include: package:flutter_lints/flutter.yaml
+
+linter:
+ 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)